trnsysGUI/segments/SegmentItemBase.py

```diff
@@ -224,9 +196,7 @@ def mouseMoveEvent(self, e):
                 self.logger.debug("Unrecognized editorMode in segmentItem mouseMoveEvent")
 
     def deleteNextHorizSeg(self, b, nextS):
-        if b:
```

Comment:
@zuckerruebe: Any idea what "b" is supposed to stand for?
This occurs further down as well

Reply:
I have changed this to keepSeg and keepPrevSeg to minimize mental effort.